tornado-drivers/spirv/src/main/java/uk/ac/manchester/tornado/drivers/spirv/graal/nodes/SPIRVIntUnaryIntrinsicNode.java

```diff
@@ -69,29 +65,21 @@ public static ValueNode create(ValueNode x, SPIRVIntOperation op, JavaKind kind)
     }
 
     private static long doCompute(long value, SPIRVIntOperation op) {
-        switch (op) {
-            case ABS:
-                return Math.abs(value);
-            case CLZ:
-                return Long.numberOfLeadingZeros(value);
-            case POPCOUNT:
-                return Long.bitCount(value);
-            default:
-                throw new TornadoInternalError("unknown op %s", op);
-        }
+        return switch (op) {
+            case ABS -> Math.abs(value);
+            case CLZ -> Long.numberOfLeadingZeros(value);
+            case POPCOUNT -> Long.bitCount(value);
+            default -> throw new TornadoInternalError("unknown op %s", op);
+        };
```

Comment:
Just for consideration, the return switch block will visit all cases since we do not break?